WalletWasabi.Fluent/Styles/TreeDataGrid.axaml

```diff
@@ -0,0 +1,84 @@
+<Styles xmlns="https://github.com/avaloniaui"
```

Comment:
Are the commented out code in this file left overs?

Reply:
No. This needs to be addressed but currently don't have solution.